domain/api/atrium-domain-api-jvm/src/main/kotlin/ch/tutteli/atrium/domain/creating/PathAssertions.kt

```diff
@@ -17,5 +17,6 @@ val pathAssertions by lazy { loadSingleService(PathAssertions::class) }
  * which an implementation of the domain of Atrium has to provide.
  */
 interface PathAssertions {
-    fun <T: Path> exists(assertionContainer: Expect<T>): Assertion
+    fun <T : Path> exists(subjectProvider: SubjectProvider<T>): Assertion
```

Comment:
Please use `Expect` instead of `SubjectProvider`, most probably I am going to deprecate SubjectProvider with 1.0.0. 
It was only introduced to ease the migration from `Assert` to `Expect`

The same applies to other places in your PR

Reply:
done